Gems/ROS2/Code/Source/GNSS/ROS2GNSSSensorComponent.cpp

```diff
@@ -124,10 +126,21 @@ namespace ROS2
             m_gnssMsg.altitude = currentPositionWGS84.m_altitude;
             m_gnssMsg.status.status = sensor_msgs::msg::NavSatStatus::STATUS_SBAS_FIX;
             m_gnssMsg.status.service = sensor_msgs::msg::NavSatStatus::SERVICE_GPS;
+
+            bool registeredPostProcessingSupport = false;
+            GNSSPostProcessingRequestBus::EventResult(registeredPostProcessingSupport, entityId, &GNSSPostProcessingRequests::IsReady);
```

Comment:
What is purpose of GNSSPostProcessingRequests::IsReady?
If there is no handler available bus event would have no effect.